test/Libraries/DynamoMSOfficeTests/ExcelTests.cs

```diff
@@ -427,7 +561,121 @@ public void CanWrite1DDataOfMixedTypesToExcelWorksheet()
             Assert.AreEqual(-90, rowList[0].Data);
 
         }
+        [Test]
+        public void Excel_MAGN6872()
+        {
+            //http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6872
+            
+            
+            string openPath = Path.Combine(TestDirectory, @"core\excel\Excel_MAGN6872.dyn");
+            ViewModel.OpenCommand.Execute(openPath);
+            Assert.AreEqual(12, ViewModel.CurrentSpace.Nodes.Count);
+        
+            var watch = ViewModel.Model.CurrentWorkspace.GetDSFunctionNodeFromWorkspace("Excel.GetDataFromExcelWorksheet");
```

Comment:
Please don't spend time testing such obsolete nodes such as `GetDataFromExcelWorksheet`. These are marked as hidden from library and are only there to support backward compatibility with older files. We should only be testing `ReadFromFile` and `WriteToFile` nodes.